README.md

```diff
@@ -24,6 +24,10 @@ We use [Netlify forms](https://docs.netlify.com/forms/setup/) to handle the form
 - We send a copy of every submission to `tyler@ideas42ventures.com`
 - Currently, we manually export a CSV for import into our mailer, Buttondown. This will change.
 
+#### Buttondown API
```

Comment:
I just had a look at the current readme. Let's tweak this a bit. Right now we have the "Email subscription form" https://github.com/ideas42ventures/ideas42ventures.com#email-subscription-form section. Let's keep that heading, but update the contents to match the new reality of our setup. We're no longer using Netlify Forms. We no longer send a copy to me, and we don't have to import the csv.

Fill out that section with what we're doing now. Imagine a new dev comes onto the project and they ask; "How is the email subscription form set up?" Give enough detail and context so they would be able to understand the setup and work on the form without needing to ask further questions.

Reply:
Updated!